src/main/java/Filters/DashboardFilter.java

```diff
@@ -20,16 +20,26 @@
 import Models.Schedule.Schedule;
 import Models.Schedule.ScheduleDAO;
 import Models.User.*;
+import static Utils.Passwords.createSaltAndHash;
 import Utils.Tables;
+import Utils.Tuple;
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+import java.util.Arrays;
+import java.util.Locale;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 @WebFilter(filterName = "DashboardFilter")
 public class DashboardFilter implements Filter {
 
     @Override
-    public void destroy() {}
+    public void destroy() {
```

Comment:
For future reference, this kind of thing should be removed from commits before making PRs